### ISSUE TITLE
Layout Grid Example 2: Polite live region feedback interrupted when adding recipient

### ISSUE THREAD

**Issue by jscholes** [w3c/aria-practices#2408]
When operating the second of the [Layout Grid Examples](https://www.w3.org/WAI/ARIA/apg/example-index/grid/LayoutGrids.html), a live region is used to confirm when a recipient has been added successfully.  However, `aria-live` on the relevant `<span>` container is set to `"polite"` rather than `"assertive"`, meaning that some screen readers will interrupt the message in favour of the announcement indicating that the input has received focus again.  This is a potential failure of WCAG SC [4.1.3 Status Messages (level AA)](https://www.w3.org/WAI/WCAG21/Understanding/status-messages).

#### Reproduce
1. With NVDA and Chrome, open https://www.w3.org/WAI/ARIA/apg/example-index/grid/LayoutGrids.html.
2. Locate the section on the page headed "Example 2: Pill List For a List of Message Recipients".
3. Type "Test" in the "Add Recipient" input.
4. Press the "Add" button.

#### Expected result

NVDA announces, "Test added. 3 recipients total.", followed by the focus change back to the "Add Recipient" input.

#### Actual behaviour
The beginning of the live region feedback is barely audible, because the speech "Add Recipient  edit  New Recipient Name  blank" interrupts it almost immediately.

CC @IsaDC
